apothecary/formulas/svgtiny/svgtiny.sh

```diff
@@ -7,7 +7,7 @@
 #
 # uses a makeifle build system
 
-FORMULA_TYPES=( "linux64" "linuxarmv6l" "linuxarmv7l" "linuxaarch64" "osx" "vs" "ios" "watchos" "catos" "xros" "tvos" "android" "emscripten" "msys2" )
```

Comment:
@danoli3 do you remember why msys2 was removed from the svgtiny formula? 
I don't think there is a msys2 / mingw  package for it. 
cc @dimitre